#ImageChessCell Msg Type Definition
uint32 x
uint32 y
float64 depth
bool isOccupied
bool isWhite
